### PR TITLE
Suppress "my password"

### PR DESCRIPTION
**Pull request by dougbu**
- false positive likely nudged by b56bb17db3ae

### CODE REVIEW

**Review by wtgodbe — APPROVED** [review]

File: CredScanSuppressions.json [hunk 1]
```diff
@@ -9,6 +9,10 @@
       "placeholder": "password",
       "_justification": "This is a fake password used in test code."
     },
+    {
+      "placeholder": "my password",
+      "_justification": "This is a fake password hash used in test code."
```
Comment (adityamandaleeka) [inline CredScanSuppressions.json:14]:
```suggestion
      "_justification": "This is a fake password used in test code."
```

**Review by adityamandaleeka — APPROVED** [review]

File: CredScanSuppressions.json (hunk 1 above)
Comment (dougbu) [reply CredScanSuppressions.json]: Thanks for the fix❕

**Review by TanayParikh — APPROVED** [review]

**Comment by dougbu** [comment]
@wtgodbe could you help out here❔ [code-mirror](https://dev.azure.com/dnceng/internal/_build?definitionId=16) builds continue to fail after merging this suppression. See https://dev.azure.com/dnceng/internal/_build/results?buildId=1696135&view=results for example.

**Comment by dougbu** [comment]
@MattGal you mean the suppressions file can't suppress an existing false positive in the repo❔ That's fairly unusable if so, especially because it's not uncommon for devs to be unaware of what may need to be suppressed.

**Comment by wtgodbe** [comment]
@MattGal I was able to push a commit to the internal repo where I just moved the suppression file to `.config`

**Comment by dougbu** [comment]
Opened #41031 in case we decide the empty commit is worth trying. Feel free to merge whenever. (I have an appointment soon.)

**Comment by dougbu** [comment]
Phew

**Pull request merged**